save-plugins/fix-and-warn-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPluginTest.kt

```diff
@@ -91,17 +88,12 @@ class FixAndWarnPluginTest {
             testFiles = emptyList(),
             fs,
             useInternalRedirections = false
-        ).execute().toList()
+        )
+        val results = fixAndWarnPlugin.execute().toList()
 
-        println("Results ${results.toList()}")
+        println("Results $results")
         assertEquals(1, results.count(), "Size of results should equal number of pairs")
-        // Check FixPlugin results
-        val tmpDir = (FileSystem.SYSTEM_TEMPORARY_DIRECTORY / FixPlugin::class.simpleName!!)
-        assertTrue("Files should be identical") {
-            // Additionally ignore warnings in expected file
-            diff(fs.readLines(tmpDir / "Test1Test.java"), fs.readLines(expectedFile).filterNot { it.contains("warn") })
-                .deltas.isEmpty()
-        }
+        // FixMe: Check FixPlugin results (assert that temporary file with formatted code has been created)
```

Comment:
I imagine, we should extract logic of temporary directory creation into a service-like component. That component can have a separate implementation for tests, when we know exactly which temporary directory has been created. For now, relying on hard-coded names makes logic too inflexible and leads to bugs in concurrent environment.